components/content/NavigationHeader.tsx

```diff
@@ -7,18 +7,53 @@ import {Title1Black, Title3Black} from 'components/text';
 import React, {useCallback} from 'react';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
 import {colorLookup} from 'theme';
-import {AvalancheCenterID} from 'types/nationalAvalancheCenter';
+import {AvalancheCenterID, AvalancheCenterWebsites} from 'types/nationalAvalancheCenter';
 
 export const NavigationHeader: React.FunctionComponent<
   NativeStackHeaderProps & {
     center_id: AvalancheCenterID;
     large?: boolean;
   }
 > = ({navigation, route, options, back, center_id, large}) => {
+  let share: boolean = false;
```

Comment:
If it's easy, could you put the "back means reset everything" behavior in a separate commit so it's easy to `git revert` if/when we decide that we don't want that? Chris seemed to agree with you that the extra jank there would be jarring for users.

Reply:
because I changed the merge queue to be squash commit, having it in a different commit wont matter. But it is a small enough bit of code that if we do need to take it out, wont be too bad since just reverting it actually wont work - we will need to add initialRoute and make sure that doesnt break other stuff.